lib/fog/openstack/baremetal.rb

```diff
@@ -34,6 +34,8 @@ class OpenStack < Fog::Service
       request :list_nodes
       request :list_nodes_detailed
       request :patch_node
+      request :set_node_power_state
+      request :set_node_provision_state
```

Comment:
Yeh, maybe they should have the noun in the request name and https://github.com/fog/fog/pull/3568 should change to have 'node_'